metricflow/test/model/parsing/test_metadata_parsing.py

```diff
@@ -102,3 +110,5 @@ def _assert_measure_metadata_is_valid(measures: Sequence[Measure]) -> None:
             f"Metadata: {measure.metadata}"
         )
         last_end_number = measure.metadata.file_slice.end_line_number
+        # if measure.name == "bookers":
```

Comment:
Apparently I did hit "add to review" on this before I submitted my review.
If this is commented out because it's no longer the correct thing to check, can we delete this? If this is commented out because, it's broken but we plan on bringing it back, can we add context about what is making this check an issue currently?

Reply:
🤦  Oops. Will remove.